src/components/auth/ForgotPasswordComp.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import PropTypes from "prop-types";
+import { SubmitButton, TextInput } from "../../views/Form/index";
+import { handleForgotPassword } from "../../redux/actions/ForgotPasswordAction";
+
+export class ForgotPassword extends Component {
+  state = {
+    user: { email: "", client_url: window.location.href }
+  };
+
+  handleChange = e => {
+    const { name, value } = e.target;
+    const { user } = this.state;
+    this.setState({
+      user: {
+        ...user,
+        [name]: value
+      }
+    });
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+    const { user } = this.state;
+    const { dispatch } = this.props;
+    dispatch(handleForgotPassword(user));
+  };
+
+  render() {
+    const { message, error } = this.props;
+    const { user } = this.state;
+    const { email } = user;
+
+    return (
+      <div className="m-auto col-sm-12">
+        <form onSubmit={this.handleSubmit} className="forgotPassword">
+          {message && (
+            <p className="text-center text-info info">
+              {message.message ? (
```

Comment:
Refactor this to 

message.message
? blahblah
:  blahblah

for readability

Reply:
I get linting errors when I do as you have requested